transaction-service/src/test/kotlin/ua/com/foxminded/restClient/gprc/TransactionGRPCServiceTest.kt

```diff
@@ -0,0 +1,48 @@
+package ua.com.foxminded.restClient.gprc
+
+import java.time.LocalDateTime
+import java.util.Currency
+import java.util.UUID
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.springframework.data.domain.Pageable
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
+import ua.com.foxminded.restClient.service.CurrencyExchangeService
+import ua.com.foxminded.restClient.service.TransactionService
+import utils.Transactions
+
+class TransactionGRPCServiceTest {
```

Comment:
write an integration test for gRPC service, please as it's entry point for clients

Reply:
done